src/coreclr/pal/inc/pal.h

```diff
@@ -2677,6 +2677,8 @@ PALIMPORT BOOL PALAPI PAL_VirtualUnwindOutOfProc(CONTEXT *context, KNONVOLATILE_
 #define PAL_CS_NATIVE_DATA_SIZE 12
 #elif defined(__FreeBSD__) && defined(__x86_64__)
```

Comment:
nit: is there HOST_X64 for consistency?

Reply:
There's HOST_AMD64, but other define's in here used `__x86_64__`